posts/2024-07-12_teal_app_development_pharmaverseadam/teal-app-development.qmd

```diff
@@ -0,0 +1,208 @@
+---
+title: "Simplifying Clinical Data Dashboards with Teal and PharmaverseAdam"
+author:
+  - name: Pavel Demin
+  - name: APPSILON
```

Comment:
Perhaps move appsilon to end of authors as it's a company? Just to align with other posts.